extensions/replication/queueProcessor/task.js

```diff
@@ -35,18 +35,14 @@ metricsProducer.setupProducer(err => {
             log.info('management init done');
 
             const replicationEndpoints = config.getReplicationEndpoints();
-            const bootstrapList = replicationEndpoints.bootstrapList
-                .filter(item => item.site === site);
-            assert(bootstrapList.length === 1, 'Invalid site argument. Site must match ' +
-                'one of the replication endpoints defined');
 
             const destConfig = Object.assign({}, repConfig.destination);
-            destConfig.bootstrapList = bootstrapList;
+            destConfig.bootstrapList = replicationEndpoints;
```

Comment:
I may be wrong, but this looks like a drastic change. Are you sure it does not break existing CRR (zenko and S3 connector), i.e. the required changes have already been applied? Otherwise with this change, it could be worth renaming the variable as well to `replicationEndpoints`.